### DISCUSSION TITLE
Create bookmark with pressing headset thrice

### DISCUSSION THREAD

**Discussion by altaheer** [advplyr/audiobookshelf#2021] [Ideas]
So I saw this pretty useful thing in the snipd app which is basically a podcast app with basically AI features. I know that the AI can't be implemented and that's not my suggestion.
What I thought was very useful is that when you press the headset button three times. It will with the help of AI bookmark the latest topic. Again. I don't expect any AI features but I always feel I want to bookmark a part of the book on the go. Wouldn't it be possible to press the headset three times and set let's say 30 seconds and when you press the headset three times it creates a bookmark of the last 30 seconds

**Comment by nichwall** [comment]
I feel like that will be headphone specific. Does this work for all headphones, or did you need to configure it? What headphones are you using?

Some people have problems with various headphones seeking/skipping wrong, so that's why I'm not sure if the "action" will be reported the same for all headphones.

**Comment by advplyr** [comment]
There is a feature request open for this https://github.com/advplyr/audiobookshelf-app/issues/548
